Bio/File.py

```diff
@@ -183,19 +191,24 @@ def peekline(self):
         return line
 
     def tell(self):
+        """Return the current position of the file read/write pointer within the File."""
         return self._handle.tell() - sum(len(line) for line in self._saved)
 
     def seek(self, *args):
+        """Set the current position at the offset specified."""
         self._saved = []
         self._handle.seek(*args)
 
     def __getattr__(self, attr):
+        """Return File attribute."""
         return getattr(self._handle, attr)
 
     def __enter__(self):
+        """Open File handle with WITH statement."""
```

Comment:
These are fine by me, 
I have some difficulty on choosing imperative mood phrases, but the more I document the more I understand how to use pydocstyle.